client/signup/steps/site-vertical/index.tsx

```diff
@@ -0,0 +1,61 @@
+import { useTranslate } from 'i18n-calypso';
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import siteVerticalImage from 'calypso/assets/images/onboarding/site-vertical.svg';
+import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+import StepWrapper from 'calypso/signup/step-wrapper';
+import { saveSignupStep, submitSignupStep } from 'calypso/state/signup/progress/actions';
+import SiteVertical from './site-vertical';
+import './index.scss';
+
+interface Props {
+	queryObject: {
+		skipSynonyms?: boolean;
+	};
+	goToNextStep: () => void;
+	isReskinned: boolean;
+	stepName: string;
+}
+
+export default function SiteVerticalStep( props: Props ): React.ReactNode {
+	const dispatch = useDispatch();
+	const translate = useTranslate();
+	const { queryObject, stepName, goToNextStep } = props;
+	const headerText = translate( 'What’s your website about?' );
+	const subHeaderText = translate( 'Choose a category that defines your website the best.' );
+
+	const submitSiteVertical = ( vertical: string ) => {
```

Comment:
Can we clarify here, is the parameter a vertical id? (In which case can we rename it to `verticalId`)

Reply:
Removed this bit and will reintroduce it when we need it